src/controller/controller.cpp

```diff
@@ -27,7 +27,7 @@ long Controller::run() {
 }
 
 bool Controller::simulate() {
-    terminalview.render(world);
+    //terminalview.render(world);
```

Comment:
same here, add a flag to go into different modes

Reply:
resolved in #28